di_website/templates/includes/partials/multi-authors.html

```diff
@@ -12,14 +14,14 @@ <h3 class="m-authors__title">
     </h3>
     <p class="m-authors__copy">
       {% if first_author %}
-        <a href="{% pageurl first_author %}?referrer=author">{{ first_author.specific.name }}</a>{% if authors|length > 0 %}, {% endif %}
+        {{ first_author.specific.name }}{% if authors|length > 0 %}, {% endif %}
       {% endif %}
       {% for author in authors %}
         {% if author.block_type == 'internal_author' %}
-          <a href="{% pageurl author.value %}?referrer=author">{{ author.value.specific.name }}</a>{% if not forloop.last %}, {% endif %}
+          {{ author.value.specific.name }}{% if not forloop.last %}, {% endif %}
         {% else %}
           {% if author.value.page %}
-            <a href="{{ author.value.page }}">{{ author.value.name }}</a>{% if not forloop.last %}, {% endif %}
+            {{ author.value.name }}{% if not forloop.last %}, {% endif %}
           {% else %}
             {{ author.value.name }}{% if not forloop.last %}, {% endif %}
           {% endif %}
```

Comment:
Nope... leave this template alone ... You're removing vital code (removing links, changing text). Just create a new one and name it `multi-authors-blog-card` or something.